client/jetpack-connect/plans.jsx

```diff
@@ -96,14 +95,8 @@ class Plans extends Component {
 	redirectToCalypso() {
 		const { canPurchasePlans, selectedSiteSlug } = this.props;
 
-		if ( selectedSiteSlug && canPurchasePlans && isEnabled( 'jetpack/checklist' ) ) {
-			return this.redirect( CALYPSO_MY_PLAN_PAGE, { 'thank-you': '' } );
-		}
-
 		if ( selectedSiteSlug && canPurchasePlans ) {
-			// Redirect to "My Plan" page with the "Jetpack Basic Tour" guided tour enabled.
-			// For more details about guided tours, see layout/guided-tours/README.md
-			return this.redirect( CALYPSO_MY_PLAN_PAGE, { tour: 'jetpack' } );
```

Comment:
I bet this was the only place using this tour so maybe we can follow up with cleaning out `client/layout/guided-tours/tours/jetpack-basic-tour/index.js` 🤔

Reply:
https://github.com/Automattic/wp-calypso/issues/33805